dev/io.openliberty.microprofile.metrics.internal.3.0_fat/fat/src/io/openliberty/microprofile/metrics/internal/fat/BadHistogramTimerConfigTest.java

```diff
@@ -0,0 +1,443 @@
+/**
```

Comment:
not sure if we need to append the ibm copy right too, need to ask. Applies for all the over tck classes.